src/System.Device.Gpio/Interop/Unix/Libc/Interop.epoll_create.cs

```diff
@@ -6,6 +6,6 @@
 
 internal partial class Interop
 {
-    [DllImport("libc", SetLastError = true)]
+    [DllImport(LibcLibrary, SetLastError = true)]
     internal static extern int epoll_create(int size);
```

Comment:
We had some comment/discussion about not using separate file for each method in other PR. And we concluded to use only one class for all methods for a certain liblrary, what your opinion about that

Reply:
If you not mind merging it into one file that would be great, I have created issue for addressing comments related to SysFs driver https://github.com/dotnet/iot/issues/146, this will be addressed with other comments, so its ok if you leave it too.

Reply:
Actually, let's not do the merge yet. I like the fact of following the same design guidelines as dotnet/corefx, so let's keep a function per file for now and discuss into the issue that @buyaa-n pointed out to see what we should do about this.